tests/run-make/raw-dylib-inline-cross-dylib/Makefile

```diff
@@ -4,15 +4,24 @@
 
 include ../../run-make-fulldeps/tools.mk
 
+# We're using the llvm-objdump instead of the system objdump to ensure it is compatible
+# with the LLVM bitcode generated by rustc.
+# Except on Windows where piping/IO redirection under MSYS2 is wonky with llvm-objdump.
+ifndef IS_WINDOWS
+OBJDUMP = "$(LLVM_BIN_DIR)"/llvm-objdump
+else
+OBJDUMP = OBJDUMP
```

Comment:
Should be `objdump` here?

Though this is an `only-windows` test, so probably having an IS_WINDOWS condition doesn't make a lot of sense.

Reply:
> Should be objdump here?

Windows is case insensitive so either way works but yeah, let's keep it in line with other tests.

> Though this is an only-windows test, so probably having an IS_WINDOWS condition doesn't make a lot of sense.

Good catch.